server/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -295,7 +304,14 @@ public final Key getRawKey() {
      * Returns <code>true</code> if this setting is dynamically updateable, otherwise <code>false</code>
      */
     public final boolean isDynamic() {
-        return properties.contains(Property.Dynamic);
+        return properties.contains(Property.Dynamic) || properties.contains(Property.OperatorDynamic);
+    }
+
+    /**
+     * Returns <code>true</code> if this setting is dynamically updateable by operators, otherwise <code>false</code>
+     */
+    public final boolean isDynamicOperator() {
```

Comment:
I wonder whether this should just be `isOperatorOnly()`
I think it would read better in `OperatorOnlyRegistry.checkClusterUpdateSettings` (because that code is really focused on _"is this an operator setting"_ rather than _"is this dynamic, but only updateable by operators"_)

Reply:
It makes senses since operator settings must be dynamic in the first place, though this assumes some understanding of the concept. But overall I agree it reads better in the call sites.